pytorch_finufft/functional.py

```diff
@@ -401,3 +401,34 @@ def finufft_type2(
     """
     res: torch.Tensor = FinufftType2.apply(points, targets, finufftkwargs)
     return res
+
+
+def finuifft_type1(
+    points: torch.Tensor,
+    values: torch.Tensor,
+    output_shape: Union[int, Tuple[int], Tuple[int, int], Tuple[int, int, int]],
+    **finufftkwargs: Union[int, float],
+) -> torch.Tensor:
+    """
+    Equivalent to :func:`~pytorch_finufft.functional.finufft_type1`,
+    but passing ``isign=1`` and dividing by the size of the output.
+    """
+    finufftkwargs["isign"] = 1
```

Comment:
This would be the inverse of the `finufft_type1` with default arguments.
Conceivably, one could also say that this function should flip whatever was put into `finufftkwargs['isign']`.
But that might be a little bit too sketchy in terms of behavior?

Could we maybe warn if `isign` was expclitly set in finufftkwargs to alert the user to the fact that this one won't be used?

Reply:
I think a warning (or even a hard error?) is more appropriate than flipping the input